tsl/src/dist_util.c

```diff
@@ -254,6 +254,26 @@ dist_util_remote_compressed_chunk_info(PG_FUNCTION_ARGS)
 	return dist_util_remote_srf_query(fcinfo, node_name, query_str->data);
 }
 
+Datum
+dist_util_remote_hypertable_index_info(PG_FUNCTION_ARGS)
+{
+	char *node_name;
+	StringInfo query_str;
+	Name schema_name, index_name;
+	/* Strict function */
+	if (PG_NARGS() != 3 || PG_ARGISNULL(0) || PG_ARGISNULL(1) || PG_ARGISNULL(2))
+		PG_RETURN_NULL();
```

Comment:
Nit: CodeCov complains about this. Not sure if it is really necessary to test, but if you want to silence CodeCov you can probably create a separate test where you define the function as non-strict and call it so that this condition is triggered.

Reply:
Yes, this is become a bit of an annoyance : I can remove this check. Since this is a strict function , it will not get triggered if called via SQL. Or be defensive and add this code and then see a bunch of code coverage errors. I don't want to add addl. unnecessary tests here. It feels like spinning wheels just because the tool points out what could be potential issues.

Reply:
I don't think you should remove the check. This seems to be used elsewhere in our code for strict functions and I think we should be consistent. I think it's fine to have tests that only intend to make coverage good, just to get the noise down, but I don't feel this is very important so you can leave it as it is as far as I am concerned.